pelutils/ds/_linecounter.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+import os
+import time
+
+import click
+import git
+import numpy as np
+import matplotlib.pyplot as plt
+from tqdm import tqdm
+
+from pelutils import get_repo
+from pelutils.ds.plots import figsize_std
+
+
+_default_extensions = ", ".join((
+    ".py", ".pyw",
+    ".c", ".cpp", ".cc", ".h",
+    ".tex",
+    ".html", ".css", ".js", ".jsx", ".ts",
+    ".java", ".kt", ".go",
+    ".rs",
+    ".r", ".m",
+))
+
+def _count(repo: git.Repo, branch: git.Head, exts: list[str]) -> tuple[np.ndarray, dict[str, np.ndarray]]:
+    """ Counts lines of all files recursively in the current working directory
+    Returns an array of commit epoch times and a dict mapping file extensions to line counts """
+    commits = list(reversed(list(repo.iter_commits())))  # List of commit from oldest to newest
+    times = np.array([c.committed_date for c in commits])
+    lines = { ext: np.zeros_like(times) for ext in exts }
+    for i, commit in enumerate(tqdm(commits)):
+        try:
+            repo.git.checkout(str(commit))
+        except git.GitCommandError:
+            continue
+        for path, __ in repo.index.entries:
```

Comment:
To _?

Reply:
Enkelt underscore kan give knas i en interaktiv interpreter, fx ved debugging, da _ betyder noget fast. Derfor er __ IMO bedre

https://stackoverflow.com/questions/24984349/is-two-underscores-favorable-over-a-single-underscore-for-a-dummy-varia